py_ecc/fields/field_elements.py

```diff
@@ -28,20 +32,21 @@
 T_FQP = TypeVar("T_FQP", bound="FQP")
 T_FQ2 = TypeVar("T_FQ2", bound="FQ2")
 T_FQ12 = TypeVar("T_FQ12", bound="FQ12")
-IntOrFQ = Union[int, T_FQ]
+IntOrFQ = Union[int, "FQ"]
```

Comment:
Not quite sure if there's benefit to the type checker, does this work the same if using `Literal` here? Just curious for my learning.
```suggestion
IntOrFQ = Union[int, Literal["FQ"]]
```

Reply:
It's not a `Literal`, it's how to get mypy to approve of the use of a class or type before it's actually defined.